DEVELOPMENT.md

```diff
@@ -0,0 +1,71 @@
+# Development
+
+Terraform provides helpful [Extending Terraform][1] documentation for best practices around writing provider code. This document provides some guidelines for working with this project.
+
+## Prerequisites:
+
+-   [Terraform][2] 0.10.x
```

Comment:
We should probably advise 0.12 right away.

Reply:
Also added a note about `tfenv`